pkg/apis/app/v2/kieapp_types.go

```diff
@@ -1012,6 +1016,20 @@ type KafkaJBPMEventEmittersObject struct {
 	CasesTopicName string `json:"casesTopicName,omitempty"`
 }
 
+// StartupStrategy
+type StartupStrategy struct {
+	// StartupStrategy to use. Default is OpenShiftStartupStrategy, option is ControllerBasedStartupStrategy.
+	StrategyName string `json:"strategyName,omitempty"`
+	// Controller Template Cache TTL to use when the Openshift StartupStrategy is choosed and BC is deployed, default is 5000
```

Comment:
Here is mentioned that this parameter is for OpenShift Startup strategy but in UI (in json file) is mentioned that this is used for Controller Based Startup Strategy. Please fix these descriptions.

Reply:
Right